{{cookiecutter.project_shortname}}/docker-compose.full.yml

```diff
@@ -92,6 +96,10 @@ services:
       - es
       - mq
       - db
+    volumes:
+      - static_data:/opt/invenio/var/instance/static
```

Comment:
Agreed. I was keeping the static files in the API and worker because the API container had them before. Maybe there was a reason I wasn't aware of (maybe it serves a static file for an error?). I will remove it and we will see!